sh/tests/hard-conditions_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
```

Comment:
You can create the required file for tests like so:
```sh
echo "echo hello word" >test-hard-conditions.sh
```

Reply:
done